phpsword/sword-bundle/1.0/manifest.json

```diff
@@ -0,0 +1,24 @@
+{
+    "bundles": {
+        "Sword\\SwordBundle\\SwordBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "copy-from-package": {
+        "install/bin/console": "%BIN_DIR%/console",
+        "install/wp/wp-config.php": "wp/wp-config.php",
+        "install/docker/": "docker/",
+        "install/docker-compose.yml": "docker-compose.yml",
+        "install/docker-compose.prod.yml": "docker-compose.prod.yml",
```

Comment:
We have a specific mechanism to add entries in Docker files that should be used instead (to avoid overriding the ones from Symfony).